rust/src/client/mod.rs

```diff
@@ -6,15 +26,19 @@ use tokio::time;
 use crate::{
     crypto::ByteObject,
     mask::model::Model,
-    participant::{Participant, Task},
-    request::Proxy,
     sdk::api::CachedModel,
     service::Handle,
     CoordinatorPublicKey,
     InitError,
     PetError,
 };
 
+mod request;
+pub use request::Proxy;
+
+pub mod participant;
+pub use participant::{Participant, Task};
+
 #[derive(Debug, Error)]
 /// Client-side errors
 pub enum ClientError {
```

Comment:
do you know if there is a way to include the contexts of `thiserror`, which are annotated to the variants anyways, into the docstrings? otherwise it's probably nice to have an extra docstring for the variants mirroring the contexts. like:
```rust
pub enum ClientError {
    #[error("failed to initialise participant: {0}")]
    /// Failed to initialize participants.
    ParticipantInitErr(InitError),

    // etc
}
```